almanachclient/v1/client.py

```diff
@@ -27,35 +27,87 @@ def get_url(self):
         return self.url
 
     def get_info(self):
+        """Display information about the current version and counts of entities in the database.
+
+        """
         return self._get('{}/{}/info'.format(self.url, self.api_version))
 
     def get_volume_types(self):
+        """List volume types.
+
+        """
         return self._get('{}/{}/volume_types'.format(self.url, self.api_version))
 
     def get_volume_type(self, volume_type_id):
+        """Get a volume type.
+
+        :arg str volume_type_id: Volume Type Uuid
+
+        :raises ClientException
+        """
         return self._get('{}/{}/volume_type/{}'.format(self.url, self.api_version, volume_type_id))
 
     def create_volume_type(self, volume_type_id, volume_type_name):
+        """Create a volume type.
+
+        :arg str volume_type_id: The Volume Type id
+        :arg str volume_type_name: The Volume Type name
+
+        :raises ClientException
+        """
         url = '{}/{}/volume_type'.format(self.url, self.api_version)
         data = {'type_id': volume_type_id, 'type_name': volume_type_name}
         self._post(url, data)
         return True
 
     def delete_volume_type(self, volume_type_id):
+        """Delete the volume type.
+
+        :arg str volume_type_id: Volume Type Uuid
+
+        :raises ClientException
+        """
         self._delete('{}/{}/volume_type/{}'.format(self.url, self.api_version, volume_type_id))
         return True
 
     def get_volumes(self, tenant_id, start, end):
+        """List volumes for a tenant.
+
+        :arg Uuid tenant_id: Tenant Uuid
+        :arg DateTime start: Y-m-d H:M:S.f
+        :arg DateTime end: Y-m-d H:M:S.f
+
+        :raises ClientException
+        """
         url = '{}/{}/project/{}/volumes'.format(self.url, self.api_version, tenant_id)
         params = {'start': self._format_qs_datetime(start), 'end': self._format_qs_datetime(end)}
         return self._get(url, params)
 
     def get_instances(self, tenant_id, start, end):
+        """List instances for a tenant.
+
+        :arg Uuid tenant_id: Tenant Uuid
+        :arg DateTime start: Y-m-d H:M:S.f
+        :arg DateTime end: Y-m-d H:M:S.f
+
+        :raises ClientException
+        """
         url = '{}/{}/project/{}/instances'.format(self.url, self.api_version, tenant_id)
         params = {'start': self._format_qs_datetime(start), 'end': self._format_qs_datetime(end)}
         return self._get(url, params)
 
     def create_instance(self, tenant_id, instance_id, name, flavor, start, image_meta=None):
+        """Create an instance for a tenant.
+
+        :arg Uuid tenant_id: Tenant Uuid
+        :arg Uuid instance_id: The instance uuid
+        :arg str name: The instance name
+        :arg Uuid flavor: The flavor uuid
+        :arg DateTime start: Y-m-d H:M:S.f
+        :arg json image_meta: The OS type, distro and version of the image
```

Comment:
image_meta is a dict, os and distro are custom keys that could be configured on Almanach server